test/parallel/test-util-inspect.js

```diff
@@ -770,23 +770,25 @@ assert.strictEqual(util.inspect(-5e-324), '-5e-324');
 // Note: Symbols are not supported by `Error#toString()` which is called by
 // accessing the `stack` property.
 [
-  [404, '404 [RangeError]: foo', '[404]'],
+  // [404, '404 [RangeError]: foo', '[404]'],
```

Comment:
Is this change breaking this test somehow? I think commenting out the test is not good

Reply:
oh whoops! it's not supposed to be commented out, i'll fix that now. i don't think it's breaking the tests tho, i think it's just flaky tests from what i can determine.